src/components/SelectComponents/Autosuggest/Autosuggest.tsx

```diff
@@ -151,6 +154,7 @@ export function Autosuggest<S>(props: Props<S>) {
                     onFocus: onFocusInput,
                     onKeyDown: (e) => handleInputKeyDown(e, highlightedIndex),
                     'data-lpignore': true,
+                    ...inputAttr,
```

Comment:
I wonder, does this sequence of attributes make sense? You could now theoretically overwrite previous attributes like `id` etc. Could that break the component?